.github/workflows/ci.yml

```diff
@@ -1,114 +1,64 @@
 name: CI
 on:
+  workflow_dispatch:
```

Comment:
What does `workflow_dispatch` enable here?

Reply:
It [enable manual runs](https://docs.github.com/en/actions/using-workflows/manually-running-a-workflow)